analyses/stackanalyses/sa.go

```diff
@@ -0,0 +1,186 @@
+package stackanalyses
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"mime/multipart"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/rs/zerolog/log"
+
+	"github.com/fabric8-analytics/cli-tools/analyses/pypi"
+	"github.com/fabric8-analytics/cli-tools/utils"
+	"github.com/jpillora/backoff"
+)
+
+// SARequestType is a argtype of RequestServer func
+type SARequestType struct {
```

Comment:
Remove SA prefix, it is already in a separate namespace.

Ditto for all places where SA prefix has been used

Reply:
Ack